src/main/java/cms/gov/madie/measure/services/ActionLogService.java

```diff
@@ -84,4 +85,30 @@ public boolean logMeasureSetAction(
             .build(),
         collection);
   }
+
+  public List<Action> findMeasureHistory(String targetId, String measureSetId) {
```

Comment:
```suggestion
  public List<Action> findMeasureHistory(String measureId, String measureSetId) {
```
Recommend naming the parameter `measureId` to help the caller better understand what's needed.

Reply:
Sure, i'll make the change